### PR TITLE
Adding default env vars file

### PR DESCRIPTION
**Pull request by wtgee**
Removing old unused file

I'm actually not sure we want the MPLBACKEND in there as it will change global settings. Makes sense for a unit on a computer but might annoy some future developer if not careful.

### CODE REVIEW

File: scripts/install/default-env-vars.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+export PANDIR=/var/panoptes
+export PANLOG=${PANDIR}/logs
+export POCS=${PANDIR}/POCS
+export PAWS=${PANDIR}/PAWS
+export PIAA=${PANDIR}/PIAA
+export MPLBACKEND=${POCS}/.matplotlib
```
Comment (jamessynge) [inline scripts/install/default-env-vars.sh]: I'm unfamiliar with this file. I don't see a .matplotlib in my clone.

File: scripts/setup-env.sh [hunk 2]
```diff
@@ -1,28 +0,0 @@
-#!/usr/bin/env bash
```
Comment (jamessynge) [inline scripts/setup-env.sh]: I think we want a file that includes these actions. Ideally a file like that used by Anaconda (?) where one executes wget to fetch the file and passes it to bash.

File: scripts/install/default-env-vars.sh (hunk 1 above)
Comment (wtgee) [reply scripts/install/default-env-vars.sh]: Sorry, meant to make a comment that this should go after #423, where I added that.

File: scripts/setup-env.sh (hunk 2 above)
Comment (wtgee) [reply scripts/setup-env.sh]: Yes, I think so. We still need to clean up the install process to make it simpler. I still want a docker image that would work well actually. But this is just  small step as I realized a) we don't have those env variables collected anywhere and b) that other script was outdated and should be removed.

**Pull request merged**